src/core/unit.c

```diff
@@ -270,6 +270,9 @@ int unit_set_description(Unit *u, const char *description) {
         if (isempty(description))
                 s = NULL;
         else {
+                if (streq_ptr(u->description, description)
```

Comment:
Missing ')'

Reply:
Not quite sure how this happened since I was cherry-picking and this patch applied cleanly IIRC. Fixed now.